README.md

```diff
@@ -286,13 +286,13 @@
   // for class name we use UpperCamelCase
   class SomeClassExample { 
     
-    // for const name we use UPPERCASE
-    const CONFIG = {
-      key: 'value'
+    // for const name we use const keyword
```

Comment:
names

Reply:
?

Reply:
`for const names we use the const keyword`
(either `a const name` or `const names`)

Reply:
https://github.com/i0natan/nodebestpractices/pull/82